cmd/admission-webhook/main.go

```diff
@@ -35,20 +36,23 @@ import (
 )
 
 const defaultGOMemlimitRatio = 0.0
+const defaultValidationScheme = "legacy"
 
 func main() {
 	var (
-		serverConfig  = server.DefaultConfig(":8443", true)
-		flagset       = flag.CommandLine
-		logConfig     logging.Config
-		memlimitRatio float64
+		serverConfig         = server.DefaultConfig(":8443", true)
+		flagset              = flag.CommandLine
+		logConfig            logging.Config
+		memlimitRatio        float64
+		nameValidationScheme string
 	)
 
 	server.RegisterFlags(flagset, &serverConfig)
 	versionutil.RegisterFlags(flagset)
 	logging.RegisterFlags(flagset, &logConfig)
 
 	flagset.Float64Var(&memlimitRatio, "auto-gomemlimit-ratio", defaultGOMemlimitRatio, "The ratio of reserved GOMEMLIMIT memory to the detected maximum container or system memory. The value should be greater than 0.0 and less than 1.0. Default: 0.0 (disabled).")
+	flagset.StringVar(&nameValidationScheme, "name-validation-scheme", defaultValidationScheme, "The name validation scheme to use ('legacy' or 'utf8'). Default: legacy.")
```

Comment:
my bad, the default value is already printed.
```suggestion
	flagset.StringVar(&nameValidationScheme, "name-validation-scheme", defaultValidationScheme, "The name validation scheme to use ('legacy' or 'utf8').")
```